projects/plugins/jetpack/extensions/plugins/launchpad-save-modal/util.js

```diff
@@ -0,0 +1,10 @@
+const FLOWS = {
```

Comment:
We should probably remove this since we want to deliver this functionality for all launchpad flows, so simply checking the launchpad enabled status should be enough.

If there are other concerns im missing, we may want to add in `write` and `build` in prep of our current work to enable launchpads for those flows as well.